client/landing/stepper/declarative-flow/internals/steps-repository/launchpad/task-helper.ts

```diff
@@ -49,6 +60,8 @@ export function getEnhancedTasks(
 
 	const firstPostPublishedCompleted = checklistStatuses?.first_post_published || false;
 
+	const planCompleted = checklistStatuses?.plan_selected || ! isStartWritingFlow( flow || null );
```

Comment:
>From my understanding is that for the other flows are using this value, the default value for this would be true. So that's why we have to write it like this? Asking to be sure that's the case.

Exactly. The other tasks assume this default value is `true`. For the `start-writing` flow we want a default value of `false`.